tests/test_system_architecture.py

```diff
@@ -0,0 +1,14 @@
+"""Testing script for system architecture classes"""
+
+import unittest
+from aind_data_schema_models.system_architecture import OperatingSystem, CPUArchitecture
+
+
+class UnitsTests(unittest.TestCase):
```

Comment:
I think the class should be `TestSystemArchitecture` 
and then line 10 is
`def test_class_contruction(self)`

just based on how the other tests are set up

Reply:
whoops, thanks for catching that